src/main/java/uk/ac/ebi/tsc/portal/api/team/controller/TeamRestController.java

```diff
@@ -195,7 +191,9 @@ public TeamResource getTeamByName(HttpServletRequest request, Principal principa
 		if(team == null){
 			throw new TeamNotFoundException(teamName);
 		}
-		return teamService.setManagerUserNames(new TeamResource(team), getToken(request));
+		TeamResource teamResource = teamService.setManagerUserNames(new TeamResource(team), getToken(request));
```

Comment:
Is the result of team creation use anywhere? If not (there might be a subsequent call to get from UI), maybe we do not need it?

Reply:
this method is getTeamByName, we will need it here